packages/app/src/apis.ts

```diff
@@ -7,6 +7,12 @@ import {
   configApiRef,
 } from '@backstage/core';
 
+import {
+  costInsightsApiRef,
+  ExampleCostInsightsClient,
+} from '@backstage/plugin-cost-insights';
+
+
```

Comment:
Every extra newline kills a kitten

Reply:
I think it's actually exponential kitten-killing as the number of extra newlines increases 🤔